scenarios/kubernetes_e2e.py

```diff
@@ -79,6 +81,13 @@ def main(args):
       '-v', '/etc/localtime:/etc/localtime:ro'
     ]
 
+    if args.docker_in_docker:
+        cmd.extend([
+          '-v', '/var/run/docker.sock:/var/run/docker.sock',
+          '-v', '%s:/go/src/k8s.io/kubernetes' % workspace,
+          '-e', 'REPO_DIR=%s' % workspace,
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/cluster/images/kubemark/Makefile#L24

Found it, you are right!